tools/cli/internal/changelog/changelog.go

```diff
@@ -42,10 +42,10 @@ type Metadata struct {
 	Revision          *load.SpecInfo //  the new spec to compare against the base
 	Config            *checker.Config
 	OasDiff           *openapi.OasDiff
-	ExceptionFilePath string
+	ExemptionFilePath string
```

Comment:
oh wow I've just noticed that the file name is Exemption not exception 🤦

Reply:
ahhaha no worries